commit 7c396b76a387bfea10ce49822bd17a1b5a8617ae (refs/changes/44/144/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2016-07-26T20:09:00+00:00 (3 years, 2 months ago)
    
    TOOLS-1497 mg should build firmware tools from github
    Reviewed by: Cody Mello <cody.mello@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index 730d741..6ad7448 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -77,7 +77,7 @@ if [[ "${JOYENT_BUILD}" == "true" ]]; then
   FIRMWARE_TOOLS_ENTRY='
   "firmware-tools": {
     "repos": [
-        {"url": "git@git.joyent.com:firmware-tools.git"}
+        {"url": "git@github.com:joyent/firmware-tools.git"}
     ],
     "public": false,
     "deps": []
